src/index.js.flow

```diff
@@ -29,10 +29,13 @@ export type BasketStore<ST> = {|
   mutator: SetState<ST>,
 |};
 
-export type BasketAction<ST, EX = *> = (
-  setState: SetState<ST>,
-  getState: GetState<ST>,
-  extraArg: EX
+export type BasketAction<ST, PR = *> = (
+  {|
+    setState: SetState<ST>,
+    getState: GetState<ST>,
+    actions: { [key: string]: BasketAction<ST, PR> },
```

Comment:
This is lazy typed but I'm not sure we can actually do better given in Jira every action is in a separate file, so if we `import actions` to use them as a type we probably trigger a cyclic dependency: action.js > actions.js > action.js